app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,6 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
-</resources>
\ No newline at end of file
+    <color name="grey">#858282</color>
+    <color name="nightRider">#353535</color>
```

Comment:
đặt tên màu rõ ràng để sau dễ gọi e

Reply:
Em tra theo trên mạng thì cái giá trị màu đó tên nó là thế ý anh

Reply:
Bạn có thể đặt tên như sau 
https://chir.ag/projects/name-that-color/#353535
```suggestion
    <color name="color_black_mine_shaft">#353535</color>
```